src/views/SystemIntake/index.tsx

```diff
@@ -42,19 +44,20 @@ export const SystemIntake = ({ match }: SystemIntakeProps) => {
   ];
   const history = useHistory();
   const [page, setPage] = useState(1);
-  const [id, setId] = useState('');
+  const [existingIntakesLoaded, setExistingIntakesLoaded] = useState(false);
   const dispatch = useDispatch();
   const formikRef: any = useRef();
+  const id = useRef<string>(uuidv4());
   const pageObj = pages[page - 1];
 
-  useEffect(() => {
-    // TODO: Make request to get system intake by id.
-    // If it doesn't exist, create a new id.
-    setId(uuidv4());
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
-
-  const initialData: SystemIntakeForm = initialSystemIntakeForm;
+  const draftIntakes = useSelector(
+    (state: AppState) => state.systemIntakes.systemIntakes
+  ).filter(intake => intake.status === 'DRAFT');
+  const initialData: SystemIntakeForm =
+    draftIntakes.length > 0 ? draftIntakes[0] : initialSystemIntakeForm;
+  if (initialData.id === '') {
+    initialData.id = id.current;
+  }
```

Comment:
@christopherhuii let me know how this looks. Setting the ID in the initial data avoids a problem where the ID would be set on save and trigger an extra API dispatch.

Reply:
I don't think the System Intake form should be reading from the System Intakes (plural) reducer.

The steps I am thinking here are:

1. App renders with initial data
2. `GET` all System Intake Forms for a user
    - It would be ideal if we can only get intakes that are `status: 'DRAFT'`, but not a big deal.
3. Once the data is received we store the system intakes in the SystemIntakes reducer.
4. We need to pick a system intake and store it in the SystemIntake reducer
    - If there are no system intakes, we load the initial data
    - Set the id
5. App reads from the SystemIntake reducer
6. App renders

Reply:
Actually, forget the part about storing one of the system intakes in the system intake reducer. there's a gap in how a user accesses their system intakes because. the backend allows the user to have multiple intakes, but the frontend only allows the user to have one.

the mismatch here is making things more difficult. so to simplify that, we can just read from the system intakes reducer and leave that for a problem to solve another day.

we should still load the intake, set the id, then render that data in formik (to avoid re-render/autosave).

Reply:
Notably, the reason I put it separate from the system_intakes part of the redux store was because it was used for the reviewer track, not the requester. If we want to reuse the single `system_intake` part of the redux store for requesters, that's ok, but I would assume the first system intake that's a draft.